shelley/pool-ranking/pool-ranking.tex

```diff
@@ -28,78 +29,119 @@
   urlbordercolor={white}
 }
 
+\setlength{\parindent}{0pt}
+\setlength{\parskip}{12pt plus 3pt minus 2pt}
+
 \maketitle
 
 \section{Introduction}
 
-The purpose of stake pool ranking is to provide a recommendation to users
-(wallets, people who own ADA, etc)
-for the stake pools which are expected to yield the greatest rewards.
-The ranking is based upon past performance but should be a useful predictor of future performance.
-
-A naive approach would be to rank pools based on the rewards that are
-currently being paid out to its delegators.
-There are at least two problems with this approach.
-
-First, the naive approach is short-sighted in the sense that it does not take into
-account what a newly formed stake pool would yield when it reaches saturation.
-This issue is explained in detail in \cite{bkks2018}. The problem as applied to
-Cardano is explained in the delegation design doc \cite[Section 5.6]{delegation_design}.
-
-The second problem is that sometimes a stake pool has the opportunity to produce blocks,
-but doesn't due to things like a bad network connection or untimely downtime. The rate at
-which a pool uses their opportunities to make blocks, which we will call hit rate, affects rewards.
-It needs to be taken into account.
-
-Our aim is to address both concerns.
+The purpose of the stake pool ranking algorithm is to provide a systematic and unique ordering for stake pools,
+based on the expected long-term rewards that each stake pool will produce.
+The pool ranking is provided to users (wallets, people who own ADA, etc.), and can be used as a single
+metric to guide delegation decisions.  The ranking is partly based on the past performance of each pool, but
+also takes into account current levels of delegation etc.  It should thus be a useful predictor of future performance.
```

Comment:
I think this could be misleading. the non-myopic ranking actually ignores the current active stake of the pool. the other components to the ranking (besides the hit-rate estimation) are the pool parameters such as cost, margin, and pledge.